### ISSUE TITLE
Players with commas in their names cannot be target

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#10983]
We got a player called "Agent Hitler, FBI" which is untargettable for any command, including /ban and /whois (so banning by ID is no option either).

At least i do not want to have an Agent Hitler on my german server, so probably that could be fixed?

**Comment by elfl0rd** [comment]
https://github.com/StrangeLoopGames/EcoSuggestions/issues/158

> For example where it needed.
> If player with inadmissible nickname join - they saved in multiple places, currences, eg.

**Comment by johnkslg** [comment]
Ive changed the system in 9.0 so everything can be specified by id and name both.

**Comment by johnkslg** [comment]
@redwyre did your other fix affect this?

**Comment by redwyre** [comment]
Yeah, you can ban via id now, but it doesn't add support for quoted strings in commands
